features/wasm-exception-handling.yml

```
@@ -0,0 +1,16 @@
+name: WebAssembly exception handling
+description: An extension to WebAssembly adding break control flow when an exception is thrown.
+spec: https://webassembly.github.io/exception-handling/js-api/
```

Again, I have no idea what this is on about. I'm going to make some wild guesses here. 😬 

```suggestion
name: Exception handling
description: Exceptions break the normal control flow of execution to represent exceptional behavior, such as an error. You can respond to the exception in JavaScript code.
spec: https://webassembly.github.io/exception-handling/js-api/
```